### PR TITLE
Add ETC support

### PR DESCRIPTION
**Pull request by passabilities**

### CODE REVIEW

File: src/ethereum/etcInfo.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+  etherscanApiServers: ['https://blockscout.com/etc/mainnet'],
+  blockcypherApiServers: [],
+  superethServers: [],
+  infuraServers: [],
+  isNestedInfuraParams: false,
+  infuraNeedProjectId: true,
+  uriNetworks: ['ethereumclassic', 'etherclass'],
+  ercTokenStandard: 'ERC20',
+  chainId: 1,
+  hdPathCoinType: 60,
```
Comment (peachbits) [inline src/ethereum/etcInfo.js:61]: Needs to be 61

File: src/ethereum/etcInfo.js [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+  checkUnconfirmedTransactions: true,
```
Comment (peachbits) [inline src/ethereum/etcInfo.js:62]: Set to false. We don't have a server for unconfirmed transactions.

File: src/ethereum/etcInfo.js [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+  iosAllowedTokens: {
```
Comment (peachbits) [inline src/ethereum/etcInfo.js:63]: Remove iosAllowedTokens as it is deprecated.

Comment (passabilities) [reply src/ethereum/etcInfo.js]: This object appears to be required by the parent object type. For now I just made it an empty object. Unless I should just remove the type and delete from both `ethInfo` and `etcInfo`

**Pull request merged**